src/app/repositories/firebase/dataset_firebase_repository.py

```diff
@@ -7,7 +9,7 @@
 
 
 class DatasetFirebaseRepository:
-    WRITE_BATCH_SIZE = 100
+    MAX_BATCH_SIZE_BYTES= 9 * 1024 * 1024
```

Comment:
Space between constant name and assignment operator. What is the reason for the calculation rather than the value?

Reply:
1024 is the size of KB so I thought it would be more readable like this instead of putting something like 9437184

Reply:
(1024 bytes = 1 KB) x (1024 = 1 MB) x 10 = 10 MB